src/sentry/static/sentry/app/views/releases/list/index.tsx

```diff
@@ -268,29 +319,31 @@ class ReleasesList extends AsyncView<Props, State> {
             <StyledPageHeader>
               <PageHeading>{t('Releases')}</PageHeading>
               <SortAndFilterWrapper>
-                <ReleaseListDisplayOptions
-                  selected={this.getDisplay()}
-                  onSelect={this.handleDisplay}
-                />
-                <ReleaseListSortOptions
-                  selected={this.getSort()}
-                  onSelect={this.handleSort}
-                />
                 <SearchBar
                   placeholder={t('Search')}
                   onSearch={this.handleSearch}
                   query={this.getQuery()}
                 />
+                <ReleaseListStatusOptions
+                  selected={activeStatus}
+                  onSelect={this.handleStatus}
+                />
+                <ReleaseListSortOptions
+                  selected={activeSort}
+                  onSelect={this.handleSortBy}
+                />
+                <ReleaseDisplayOptions
+                  selected={activeDisplay}
+                  onSelect={this.handleDisplay}
+                />
```

Comment:
Do we want to change the order on mobile so that it looks like this?

![image](https://user-images.githubusercontent.com/9060071/100748367-895b8000-33e3-11eb-8a5b-706450cc8116.png)

vs.

![image](https://user-images.githubusercontent.com/9060071/100748508-b0b24d00-33e3-11eb-8b9d-4f8febf12219.png)